.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
+
+before_install:
+  - npm install -g dredd
+  - cd cucumber
+  - bundle install
+
+script:
+  - cd cucumber
+  - bundle exec cucumber
```

Comment:
There are no Go tests at all which is why the CI is failing. If you want to test only this, you can probably use ruby.